docs/ROADMAP.md

```diff
@@ -49,45 +49,59 @@ Non-Technical Requirements for v0.1:
   * License Agreement
 * Account information/access from WHO to submit app on behalf of their accounts (also needed for code signing) ([#20](https://github.com/WorldHealthOrganization/app/issues/20) :warning:)
 
-Other notes on v0.1:
+#### Other Notes
 
 * We switched to the Flutter client framework for 0.1, and going forward.
 
-### v0.2
+### v1.0
 
-**Target Date:** March 27, 2020
+**Goal:** Match or exceed the existing content in the WHO WhatsApp Bot presented idiomatically with native look-and-feel, with dynamically-updatable content localized to at least one other language besides English and ability to deliver push notifications to users.
 
-**Goal:** v0.2 is not a separate milestone, it simply whatever is in the repo by EOD Friday for preview by the WHO.
+**Target Date:** Monday, March 30, 2020 - transmission of app to WHO team for app store submission by EOD PDT
 
-### v1.0
+**GitHub Milestone:** https://github.com/WorldHealthOrganization/app/milestone/4
 
-**Target Date:** March 30, 2020
+#### Technical Features / Requirements
 
-**Goal:** In one sentence: v1.0 is a native look-and-feel, dynamically-updatable “port” of the WhatsApp bot content on iOS/Android, with information able to be served specific to your country and language (with at least 2 languages actually implemented), and the ability to push notifications in the future.
+* Content parity with the [WHO WhatsApp Bot](https://api.whatsapp.com/send?phone=41225017615&text=hi&source=&data=)
+* Native look-and-feel, designed to present content on mobile screens
+* Localized content translated to at least one other language besides English
+* Offline functionality with default content baked into the app bundle
+* Dynamically updated content served from the cloud so users don't need to update the app to receive content updates
+* Localized push notification capabilities
+  * _(Needs Discussion)_ What are the localization requirements for v1.0? What level of granularity will we need in region (e.g. language, continent, country, city, etc)? How will the user choose their location(s) for push delivery?
+* Instrumentation capabilities for understanding aggregated usage information to be used for product improvement, complying with all applicable laws & regulations and app policies
+* _(Needs Discussion)_ Distribution pipeline using [fastlane](https://fastlane.tools/) for integration with WHO's existing app distribution process
```

Comment:
Analytics should be included in there as a feature, very difficult to know what we're doing without that

Reply:
I'd used the "Instrumentation" terminology to include analytics, but I added analytics to that bullet point for readability 👍